syntax = "proto3";
import "GSEStateFlags.proto";
import "GSEErrors.proto";

package payload;

message GSE_TO_GCS_DATA_2 {
    optional common.GSEStateFlags state_flags = 1;

    optional float internal_temp = 2;
    optional float wind_speed = 3;
    optional int32 gas_bottle_weight_1 = 4;
    optional int32 gas_bottle_weight_2 = 5;
    optional float analog_voltage_input_1 = 6;
    optional float analog_voltage_input_2 = 7;
    optional float additional_current_input_1 = 8;
    optional float additional_current_input_2 = 9;

    optional common.GSEErrors error_flags = 10;
}
